chainer/functions/connection/linear.py

```diff
@@ -69,6 +69,81 @@ def backward(self, indexes, grad_outputs):
         return ret
 
 
+class LinearGradData(function_node.FunctionNode):
+
+    def __init__(self, linear):
```

Comment:
Please remove the constructor if it is unused.

Reply:
I will remove it, thanks